doc/architecture/decisions/0003-bulk-load-gp-data.md

```diff
@@ -0,0 +1,23 @@
+# 3. Build Image with GP Data
+
+Date: 2017-06-15
+
+## Status
+
+Accepted
+
+## Context
+
+In the early stages of ES adoption (which predates the development of ETL processes to automatically build the GP JSON data file) the GP data mas manually added to this repo. The data is in a format optimised for loading into MongoDB. Whilst Elastic Search (ES) can import documents singly in the format it takes some considerable time. By re-shaping the data to that required by the bulk import API of ES then the import can happen in a matter of seconds rather than 10's of minutes otherwise.
```

Comment:
I think the context can be simplified quite a bit to something like `Other work within the same domain has created a JSON file containing all GP practices that need to be searched and exposed via ES. This datafile was originally created to be loaded into MongoDB.` (or something equally or more succinct.